src/main/java/Window/Controller.java

```diff
@@ -33,15 +33,13 @@
     @FXML private void open(final ActionEvent event) throws IOException {
```

Comment:
Method name is still too general. I suggest changing it to something like "openNewFile".